test_echo.py

```diff
@@ -0,0 +1,14 @@
+from echo_client import clientmessage
+import pytest
+
+
+def test_ping():
+    assert clientmessage("this is a test") == "this is a test"
+
+
+def test_big():
+    assert clientmessage("this is a string that is much longer than the 32 bites that it has to force it to loop") == "this is a string that is much longer than the 32 bites that it has to force it to loop"
+
```

Comment:
This line of code is far longer than the pep8 standard of 79 chars.  Figure out a way to make it shorter.